overlord/servicestate/servicestate.go

```diff
@@ -49,9 +49,42 @@ type Instruction struct {
 
 type ServiceActionConflictError struct{ error }
 
+type ResolvedAppInfo struct {
+	snap.AppInfo
```

Comment:
either way as this is a thin decorating wrapper this would usually be *snap.AppInfo